packages/docusaurus/src/webpack/plugins/WaitPlugin.ts

```diff
@@ -5,20 +5,27 @@
  * LICENSE file in the root directory of this source tree.
```

Comment:
For some reason, copyUntypedFiles kept failing on this file locally, so I just ported it to TypeScript.